src/index.js

```diff
@@ -13,7 +13,6 @@ $(document).ready(function (){
     render(<Models />, document.getElementById('models-container'));
     
     var content = require('./populate-content');
-    var contentScroller = require('./content-scroller');
     var cardWidget = require('./card-widget');
```

Comment:
Why remove require('./content-scroller')?

Reply:
The content scroller functionality is now performed by the carousel.js component